magento-engcom/msi#1454 [MFTF] Automated MSI Test Cases - Part 5

ivy00johns opened the pull request:
### Description
- This PR contains:
    - 3 additional Tests for MSI.

### Tests
1. [Configuration-Sources] Editing Source Code of saved Source is not allowed: https://app.zenhub.com/workspace/o/magento-engcom/msi/issues/1471
2. [Catalog-Products-Simple Product] Admin user applied 'Use decimals' to Simple product in Advanced inventory on product page: https://app.zenhub.com/workspace/o/magento-engcom/msi/issues/1472
3. [Catalog-Products-Virtual Product] Manage Stock on Configuration page turned off for Virtual Product in admin: https://app.zenhub.com/workspace/o/magento-engcom/msi/issues/1474

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

ivy00johns committed:
magento-engcom/msi#1454 [MFTF] Automated MSI Test Cases - Part 5

- Adding 3 additional Test Cases.

magento-cicd2 commented:
[![CLA assistant check](https://cla.dev.magento.com/pull/badge/not_signed)](https://cla.dev.magento.com/magento-engcom/msi?pullRequest=1483) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-engcom/msi?pullRequest=1483) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: ivy00johns<br/>:x: John Stennett<hr/>**John Stennett** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

ivy00johns committed:
magento-engcom/msi#1551 [MFTF] MSI Test Updates

- Updating Test Annotations to make them compatible with MFTF v2.3.x

ivy00johns committed:
Merge branch '1551-MFTF-MSI-Test-Updates' into 1454-MSI-MFTF-Test-Cases-5

ivy00johns committed:
magento-engcom/msi#1497

- Adding "multi_mode" group.
- Updating Test annotations to make the Tests compatible with MFTF v2.3
- Replacing "VirtualProduct" with "VirtualMsiProduct".
